core/src/main/scala/vault/http/Http4sVault.scala

```diff
@@ -154,12 +154,12 @@ final class Http4sVaultClient(
     authBackendPrefix.map(_ + cn).getOrElse(cn)
 
   def createPKIRole(cpkir: CreatePKIRole): IO[Unit] = {
-    val engine = kubernetesAuthEngineName(cpkir.engineName)
-    reqVoid(IO.pure(Request(POST, v1BaseUri / engine / "roles" / cpkir.roleName)))
+    val pkiPath = cpkir.pkiPath.getOrElse("pki")
```

Comment:
happy to move this instantiation higher up in the hierarchy of config initialization if it makes more sense~